### ISSUE TITLE
Initialization of OCR layers

### ISSUE THREAD

**Issue by carhartt21** [HRNet/HRNet-Semantic-Segmentation#132]
Hello, 

I was wondering why the layers of the OCR module are excluded from the `init_weights` function and whats the proper why to initialize them. 
I am trying to do my own implementation of the OCR module but somehow struggle with that point. 
After training the weights of the OCR layer are relatively high (~1), which seems to lead to faulty results.

Regards

**Comment by PkuRainBow** [comment]
@carhartt21 We just use the default initialization (following the Pytorch) for all the layers within the OCR module.

**Comment by carhartt21** [comment]
@PkuRainBow Thanks for replying. I think the problem I had was due to the learning rate. 
I was using the settings from the config files in the experiments folder. But I didn't notice that the lr of the OCR module is hardcoded to be 10x the lr of the backbone in `def adjust_learning_rate`.
I adjusted my implementation accordingly and the weights of the OCR layers are now much lower.

**Comment by PkuRainBow** [comment]
@carhartt21 Great~

**Comment by seekFire** [comment]
@carhartt21 you mean hrnet & ocr have different learning rate when training the backbone(hrnet + ocr), right?

**Comment by PkuRainBow** [comment]
@seekFire For the experiments on Cityscapes, ADE20K & LIP, we adopt the same learning rates for both HRNet and OCR.

For the experiments on Pascal-VOC, COCO-Stuff, Pascal-Context, applying 10x larger learning rates on the OCR head brings better performance.

**Comment by seekFire** [comment]
@PkuRainBow Okay! So I just wonder what's the basis for doing this? Is it based on the size of the dataset?

**Comment by PkuRainBow** [comment]
@seekFire In fact, we originally try to verify "10x larger learning rates on the context head" over different datasets following the previous works and find that sometimes it works while other times it doesn't work.

**Comment by seekFire** [comment]
@PkuRainBow It may seem to be based on some specific features of the dataset...Anyway thank you for your explanation!
